docs/modules/ROOT/partials/proc-persistence-kafkasql-plain.adoc

```diff
@@ -1,5 +1,5 @@
-[id="registry-persistence-kafka-streams-plain"]
-= Configuring plain Kafka Streams storage with no security
+[id="registry-persistence-kafkasql-plain"]
+= Configuring plain Kafka storage with no security
```

Comment:
The example at the start of this file shows a Kafka version of 2.5.0. This should be Kafka 2.7.0 (used by the supported version of AMQ Streams 1.7)